### ISSUE TITLE
Theming color gets ignored if you activate fonts/themes in accessibility-app

### ISSUE THREAD

**Issue by MariusBluem** [nextcloud/server#10058]
@juliushaertl @skjnldsv 🤔 

**Nextcloud:** 14.0 build 2018-06-29
**browser:** safari 12.0 on macOS Mojave beta 2

**Comment by MariusBluem** [comment]
> GitMate.io thinks possibly related issues are #7401 (Theming on light color), #9479 (Theming App Problem), #7071 (Missing icons at custom theme-color (Theming-App)), #7100 (Theming app and ImageMagick), and #985 (Should the primary button have the color of the theming app?).

![xfk8y](https://user-images.githubusercontent.com/19711361/42105170-ab71fb20-7bcf-11e8-93b7-ff9c72423b9f.jpg)

**Comment by rullzer** [comment]
Should be fixed in master

**Comment by MorrisJobke** [comment]
I tested and it works on master
